eZ/Publish/Core/Persistence/Legacy/User/Gateway/DoctrineDatabase.php

```diff
@@ -55,9 +55,6 @@ public function createUser(User $user)
                 $this->handler->quoteColumn('login'),
                 $query->bindValue($user->login)
```

Comment:
> Shouldn't we here - when creating a new user - also make $user->login lowercase (as it was done for login_normalized before)?

no, we want to retain the login name user entered

> Unless I'm missing something or it is handled elsewhere...
see schema changes for postgres, and it should be more clear.